app/views/layouts/_bottombar.html.erb

```diff
@@ -1,10 +1,10 @@
-<div class="col-md-6">                      
+<div class="col-md-6">
   <p class="copyright-space">
-  2015 &copy; F-teknologsektionen inom TLTH.
-  </p>         
+    <%= Time.current.year %> &copy; F-teknologsektionen inom TLTH.
+  </p>
+</div>
+<div class="col-md-6">
+  <p class="copyright-space pull-right">
+  <%= t('footer.control_the_web') %>: <%= render 'contacts/kontakt', name: 'Spindelmän' %>
```

Comment:
Our style is to user English for variable and file names. contacts/kontakt seems a bit out of place.